app/config/AWS.scala

```diff
@@ -1,47 +1,50 @@
 import software.amazon.awssdk.awscore.client.builder.AwsClientBuilder
-import software.amazon.awssdk.regions.{Region => RegionV2}
+import software.amazon.awssdk.regions.Region
 import software.amazon.awssdk.services.dynamodb.DynamoDbClient
+import software.amazon.awssdk.services.s3.model.GetObjectRequest
 import software.amazon.awssdk.services.s3.{S3AsyncClient, S3Client}
 
 import scala.jdk.CollectionConverters._
+import scala.util.Try
 
 case class InstanceTags(stack: String, app: String, stage: String)
 
 class AWS {
-  val region = Region.getRegion(Regions.EU_WEST_1).getName
+  val region = Region.EU_WEST_1
   val profile = "composer"
 
-  val composerAwsCredentialsProvider = new AWSCredentialsProviderChain(
-    new EnvironmentVariableCredentialsProvider(),
-    new SystemPropertiesCredentialsProvider(),
-    InstanceProfileCredentialsProvider.getInstance(),
-    new ProfileCredentialsProvider(profile)
-  )
-
-  val asgClient: AmazonAutoScaling = AmazonAutoScalingClientBuilder.standard().withRegion(region).withCredentials(composerAwsCredentialsProvider).build()
-  val ssmClient: AWSSimpleSystemsManagement = AWSSimpleSystemsManagementClientBuilder.standard().withRegion(region).withCredentials(composerAwsCredentialsProvider).build()
-  val sesClient: AmazonSimpleEmailService = AmazonSimpleEmailServiceClientBuilder.standard().withRegion(region).withCredentials(composerAwsCredentialsProvider).build()
 
   private val v2CredentialsProvider = AwsCredentialsProviderChain.builder.credentialsProviders(
-    ProfileCredentialsProviderV2.builder.profileName(profile).build,
-    InstanceProfileCredentialsProviderV2.builder.asyncCredentialUpdateEnabled(false).build,
-    EnvironmentVariableCredentialsProviderV2.create()
+    ProfileCredentialsProvider.builder.profileName(profile).build,
+    InstanceProfileCredentialsProvider.builder.asyncCredentialUpdateEnabled(false).build,
+    SystemPropertyCredentialsProvider.create(),
+    EnvironmentVariableCredentialsProvider.create()
   ).build
 
-  private def setup[B <: AwsClientBuilder[B, _]](builder: B): B =
-    builder.credentialsProvider(v2CredentialsProvider).region(RegionV2.EU_WEST_1)
+  val asgClient: AutoScalingClient = AutoScalingClient.builder.region(region).credentialsProvider(v2CredentialsProvider).build()
+  val ssmClient: SsmClient = SsmClient.builder().region(region).credentialsProvider(v2CredentialsProvider).build()
+  val sesClient: SesClient =  SesClient.builder().region(region).credentialsProvider(v2CredentialsProvider).build()
+
+  object PandaHelpers {
+    def forAwsSdkV2(anonS3Client: S3Client, bucket: String): S3BucketLoader =
+      (key: String) => {
+        anonS3Client.getObject(
+          GetObjectRequest.builder().bucket(bucket).key(key).build()
+        )
+      }
+  }
```

Comment:
This can be removed as it is provided by the Panda auth library: `S3BucketLoader.forAwsSdkV2`